hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMPerformanceMetrics.java

```diff
@@ -299,4 +309,16 @@ public MutableRate getGetObjectTaggingAclCheckLatencyNs() {
   public void addGetObjectTaggingLatencyNs(long latencyInNs) {
     getObjectTaggingAclCheckLatencyNs.add(latencyInNs);
   }
+
+  public void setLatencyDirectoryDeletingService(long latencyInNs) {
```

Comment:
As we are setting the latency in ms, let's use the variable denoting ms,
```suggestion
  public void setLatencyDirectoryDeletingServiceMs(long latencyInMs) {
    latencyDirectoryDeletingService.set(latencyInMs);
}

```

Reply:
Thanks for reviewing! I have made the changes as you have suggested along with some more changes to make the naming clearer